OpenMEEG/tools/vtp_to_meshes.cpp

```diff
@@ -49,17 +49,17 @@ int main( int argc, char **argv) {
 
     print_version(argv[0]);
 
-    command_usage("Convert meshes into a single VTP file.");
-    std::string input;
-    std::string output;
+    command_usage("Convert a single VTK/VTP into meshes.");
+    const char * input;
+    const char * output;
```

Comment:
-This "Convert meshes into a single VTP file"  was not the correct message.
- string Vs char * : command_option function requires "char *" and not "string". This worked somehow, but cause a seg_fault in certain cases (try "om_vtp_to_meshes -h" for ex.)